internal/connector/pingone/sso/pingone_sso_connector.go

```diff
@@ -59,6 +59,7 @@ func (c *PingOneSSOConnector) Export(format, outputDir string, overwriteExport b
 		resources.PopulationDefault(&c.clientInfo),
 		resources.Resource(&c.clientInfo),
 		resources.ResourceAttribute(&c.clientInfo),
+		resources.ResourceSecret(&c.clientInfo),
```

Comment:
The resource also needs to be added to the pingone_sso_connector_test.go file for terraform plan integration testing.